src/test/java/com/wikia/webdriver/common/templates/NewTestTemplateCore.java

```diff
@@ -63,12 +63,18 @@ protected void prepareURLs() {
   }
 
   protected void startBrowser() {
-    driver =
-        registerDriverListener(NewDriverProvider.getDriverInstanceForBrowser(Configuration
-            .getBrowser()));
+    driver = registerDriverListener(
+        NewDriverProvider.getDriverInstanceForBrowser(Configuration.getBrowser()));
+  }
 
-    driver.manage().deleteAllCookies();
-//    driver.manage().timeouts().pageLoadTimeout(100, TimeUnit.SECONDS);
+  protected void setWindowSize() {
+    if (!Configuration.getBrowser().contains("ANDROID")) {
+      if (Configuration.getBrowserSize() != null) {
+        driver.manage().window().setSize(Configuration.getBrowserSize());
```

Comment:
I would add `Assertion` that actually we have such size which we wanted or warning at least if the size is differs.
Since for example we may set browser size which is not supported by VM.